<launch>
  <param name="use_sim_time" value="true"/>

  <arg name="bag_file_name" value="0508/output.bag"/>
  <node pkg="rosbag" type="play" name="rosbag_play" output="screen" args="--clock /home/sun/RosProjects/Data/$(arg bag_file_name)" />

  <param name="robot_description" textfile="$(find stair_mapping)/urdf/hex_mini_model.urdf" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

  <param name="imu_pose_calibrate_r" value="-0.01"/>
  <param name="imu_pose_calibrate_p" value="-0.04"/>
  <param name="imu_pose_calibrate_y" value="0.0"/>
  <param name="need_ground_truth" value="false"/>
  <param name="display_process_details" value="false"/>
  <node name="robot_tf_broadcast_node" pkg="stair_mapping" type="robot_tf_broadcast_node" />
  <node name="stair_model_match_node" pkg="stair_mapping" type="stair_model_match_node" output="screen"/>
  <!--
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find stair_mapping)/rviz/offline_data_view.rviz"/>
  -->
</launch>
